_layouts/default.html

```diff
@@ -0,0 +1,76 @@
+{%- if page.app-bar.trailing-control.target -%}
+    {%- capture app-bar-trailing-control-url -%}
+        {%- case page.app-bar.trailing-control.target -%}
+            {%- when 'github' -%}
+                https://GitHub.com/{{ site.social.github }}
+        {%- endcase -%}
+    {%- endcapture -%}
+
+    {%- capture app-bar-trailing-control-icon -%}
+        {%- case page.app-bar.trailing-control.target -%}
+            {%- when 'github' -%}
+                <i class="fab fa-github fa-2x"></i>
+        {%- endcase -%}
+    {%- endcapture -%}
+{%- endif -%}
+<!DOCTYPE html>
+<html class="brightness-dark">
+<head>
+{% include header-meta.html %}
+
+{% include header-stylesheets.html %}
+
+{% include header-scripts.html %}
+
+
+
+<!-- See also the git repository at https://GitHub.com/BenLeggiero/BenLeggiero.me -->
+</head>
+
+<body>
+<header class="app-bar">
```

Comment:
Maybe this should be moved to [BlueHuskyStudios/jekyll-components](https://github.com/BlueHuskyStudios/jekyll-components)

Reply:
https://github.com/BlueHuskyStudios/jekyll-components/issues/3